specs/swagger.yml

```diff
@@ -969,26 +998,147 @@ paths:
           required: true
           type: string
       responses:
-        204:
+        '204':
           description: Success
           headers:
             X-Timestamp:
               type: number
               description: FogController server timestamp
-        400:
+        '400':
           description: recieverId Is Not Valid
-        401:
+        '401':
           description: Not Authorized
-        404:
+        '404':
           description: Not Found
-      
-
-
+  '/iofog/proxy':
+    post:
+      tags:
+        - Proxy
+      description: Creates ioFog node proxy config
+      operationId: createIOFogNodeProxyConfig
+      parameters:
+        - in: header
+          name: Authorization
+          description: User Token
+          required: true
+          type: string
+        - in: body
+          name: fogNodeInfo
+          required: true
+          schema:
+            $ref: '#/definitions/CreateIOFoNodeProxyRequestBody'
+      responses:
+        '201':
+          description: Created
+          schema:
+            $ref: '#/definitions/IOFogNodeProxyResponse'
```

Comment:
response is not required. user already has the fogId (passed as path parameter)

Reply:
fixed